sdk/Metrics/Meter.php

```diff
@@ -8,5 +8,56 @@
 
 class Meter implements API\Meter
 {
-    // Temp stub
+    /**
+     * @var string $name
+     * @var string $version
+     */
+    private $name;
+    private $version;
```

Comment:
No big deal but I prefer to use protected properties instead of private.
The reason behind it is that a developer may need to extend Meter's functinality. In order to do that he will extend this class. The extended class will miss these 2 properties. But actually these properties will be dynamically created in runtime. This MAY lead to some unexpected things